### ISSUE TITLE
gretty webAppCopy also affects the war task

### ISSUE THREAD

**Issue by levsa** [akhikhl/gretty#193]
I've added test resources to the webapp using gretty's webAppCopy:

```gradle
    webappCopy {
        from "$projectDir/src/test/resources"
        include "**/*"
    }
```

But this affects the war task from the war plugin as well, i.e., I get the test resources in the production webapp. Is this by design? It would be nice and make testing easier if this only affected the gretty tasks.

**Comment by akhikhl** [comment]
yes, this was specifically requested by other Gretty users. I'll try to introduce control over it, maybe some property.

**Comment by levsa** [comment]
That would be great, thanks! My current, not so nice solution is an if-statement around webappCopy that checks the tasks given on the command line :-/

**Comment by Dispader** [comment]
Hey, there— if I could ask, have you found a better workaround— or could I see your solution?  I seem to be having the same problem.

**Comment by NielsUll** [comment]
Any news on this? It really breaks the plugin for us. 

We want to build normal war files for deployment on jboss, but want to test locally on jetty, 
To do this, we need to change our spring contextConfigLocation to a jetty-specific one.

**Comment by baldervanx** [comment]
Still nothing on this? I have some filtering that I only want to apply when running locally withing gretty, but not when deploying to production. With my limited gradle knowledge I can't find any workaround for this, like disabling gretty when building for production.
